docs/library/uarray.rst

```diff
@@ -27,3 +32,56 @@ Classes
 
         Append new elements as contained in *iterable* to the end of
         array, growing it.
+
+    .. method:: decode(encoding="utf-8", errors="strict")
+
+        Return a string decoded from *self*. 
+        Default *encoding* is 'utf-8'. 
+        *errors* may be given to set a different error handling scheme. 
+        The default for errors is 'strict', meaning that encoding errors raise a ``UnicodeError``. 
+        Other possible values are 'ignore' and 'replace'.
+        **Note:** ``decode`` is unique to MicroPython and behaves like ``array.tobytes().decode(...)`` does in Python
+        (but  is more efficient because it does not creat an intermediates ``bytes`` object).
```

Comment:
I don't think `decode` was ever intended to exist on array instances.  It's there only for `bytearray` which is (for efficiency of code-size) implemented with the same code as other array type-codes.

So I'd suggest to never document this method on array because it shouldn't be used and may one day disappear.

Reply:
Would it be better to mark the method as deprecated? I suggest this because I discovered the method using `dir` and others might too.